pywren/version.py

```diff
@@ -18,7 +18,7 @@
 # we're following the version number guidelines
 # from https://www.python.org/dev/peps/pep-0386/
 
-__version__ = "0.4.0"
+__version__ = "0.4.0rc0"
```

Comment:
should this be rc0 ?

Reply:
Not that I know of I'm not sure what our rc policy is but, since creating/moving tags in git is annoying, I'm going to create the 0.4 tag only when things are final.

Reply:
Just a warning that you shouldn't push to pypi without updating this ?